public/app/core/utils/dag.ts

```diff
@@ -252,7 +252,7 @@ export class Graph {
     return new Edge();
   }
 
-  getNode(name: string): Node {
+  getNode(name: string): Node | undefined {
```

Comment:
This should have been typed like this a long time ago, I was seeing crashes that TypeScript could have caught.